src/js/views/programs/program/flow/action-item.hbs

```diff
@@ -12,7 +12,7 @@
     <span class="table-list__meta" data-due-region></span>{{~ remove_whitespace ~}}
     <span class="table-list__meta">
       {{#if hasForm}}
-        <button class="js-form button-secondary--compact">{{far "square-poll-horizontal"}}</button>
+        <button class="button-secondary--compact" disabled>{{far "square-poll-horizontal"}}</button>
```

Comment:
I switched the form icon for action list items to be `disabled`.

Instead we could also just show a form icon without button styling or keep the button enabled and open the action sidebar when the form button is clicked.